backend/app/routers/auth.py

```
@@ -1,19 +1,40 @@
+)
 from fastapi.responses import JSONResponse
 from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
 from sqlalchemy.orm import Session
 from datetime import timedelta
 from typing import Annotated, List, Optional
 from passlib.context import CryptContext
-from user_management.models import User, Order, OrderItem, Bill  # adjust as needed
-from user_management.schemas import UserCreate, Token, LoginRequest, ForgotPasswordRequest, ResetPasswordRequest
-from user_management.utils import auth_utils, create_access_token, verify_token, create_email_verification_token, send_verification_email, verify_reset_token, create_reset_token, send_reset_email
-from user_management.database import get_db
+from app.models import User, Order, OrderItem, Bill  # adjust as needed
```

Ruff (0.8.2)

`app.models.Order` imported but unused

Remove unused import

(F401)